### DISCUSSION TITLE
Suggestion: add a search feature on the backpack

### DISCUSSION THREAD

**Discussion by Alias1XP** [ScratchAddons/ScratchAddons#5151] [Ideas]
hi, i have a suggestion, please add a search feature on the backpack, i have many things in my backpack, and when i go to get something it takes a long time, i have to constantly look around for it, it would be much easier with a search feature on the backpack, thanks for reading.

**Comment by BroJac5246** [comment]
Feel free to create an issue that suggests this :)

**Reply by Alias1XP** [reply]
?

**Reply by BroJac5246** [reply]
Go [here](https://github.com/ScratchAddons/ScratchAddons/issues/new?assignees=&labels=new+addon%2Cscope%3A+addon%2Ctype%3A+enhancement&template=--new-addon.yml), select "New Addon," and fill out the form. It'll create a new 'issue' suggesting the addon.
